app/controllers/users_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def index
+    @users = User.all
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+
+    if @user.update(user_params)
+      redirect_to edit_user_path
```

Comment:
- [ ] `edit_user_path` needs a user parameter. 
**Hint** You can check the generated route in the console. If you typed `app.edit_user_path` in the console it will give an error missing required key. So you need to pass the `@user` to `edit_use_path` to make it work.